app/controllers/TaskController.scala

```diff
@@ -83,7 +83,7 @@ Expects:
  - As Form data:
    - taskTypeId (string) id of the task type to be used for the new tasks
    - neededExperience (Experience) experience domain and level that selects which users can get the new tasks
-   - openInstances (int) if greater than one, multiple instances of the task will be given to users to annotate
+   - pendingInstances (int) if greater than one, multiple instances of the task will be given to users to annotate
```

Comment:
would it make sense to rename this to `initialPendingInstances`? or only `instancesCount`? I find this `pending` here a bit confusing :thinking:

Reply:
if create task where the only place where this was used, I’d agree, but the same class is also used when updating a task, and there pending instances makes more sense. So I’d rather leave it as is here as well

Reply:
Ok, I see :+1: